### PR TITLE
fix/morning brew newsletter parsing

### PR DESCRIPTION
**Pull request by sywhb**
- Add test page
- Generate newsletter test pages in a subdirectory
- Generate morning brew newsletter test page from email content

### CODE REVIEW

**Comment by jacksonh** [comment]
nice!

File: apple/Omnivore.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1439,7 +1439,7 @@
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = QJF2XZ86HB;
+				DEVELOPMENT_TEAM = MHM9HW2S8F;
```
Comment (jacksonh) [inline apple/Omnivore.xcodeproj/project.pbxproj:1442]: Can you remove this file from the PR? Apple just does this automatically.

**Review by jacksonh — CHANGES_REQUESTED** [review]

File: apple/Omnivore.xcodeproj/project.pbxproj (hunk 1 above)
Comment (sywhb) [reply apple/Omnivore.xcodeproj/project.pbxproj:1442]: Yes, sorry I didn't notice it

**Review by jacksonh — APPROVED** [review]

**Review by jacksonh — APPROVED** [review]

**Comment by jacksonh** [comment]
@sywhb ok to merge this one?

**Comment by sywhb** [comment]
> @sywhb ok to merge this one?
Merged!

**Pull request merged**